src/main/java/fi/metatavu/dcfb/server/rest/ItemsApiImpl.java

```diff
@@ -119,13 +138,8 @@ public Response createItem(Item payload) throws Exception {
       return createBadRequest(String.format("Invalid location %s", payload.getLocationId()));
     }
     
-    Currency priceCurrency = getPriceCurrency(payload.getUnitPrice());
-    if (priceCurrency == null) {
-      return createBadRequest(String.format("Invalid currency %s", payload.getUnitPrice().getCurrency()));
-    }
-    
-    Boolean allowPurchaseContactSeller = payload.getPaymentMethods().isAllowContactSeller();
-    Boolean allowPurchaseCreditCard = payload.getPaymentMethods().isAllowCreditCard();
+    Boolean allowPurchaseContactSeller = payload.getPaymentMethods() != null ? payload.getPaymentMethods().isAllowContactSeller() : false;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the literal "false" boolean value. ([squid:S1125](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1125))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=47&issues=AWZVH_tLBfefbO9HrBx2&open=AWZVH_tLBfefbO9HrBx2)

[ISSUE_KEY]: AWZVH_tLBfefbO9HrBx2